index.d.ts

```diff
@@ -21,20 +21,29 @@ export interface Nodeback<E, R> {
   (err: E | null, value?: R): void
 }
 
-export interface ConcurrentFutureInstance<L, R> {
+declare const $functor: unique symbol
```

Comment:
I named this after the interface it's used in. But in theory I could use this symbol in other similar interfaces, right? Maybe I should give it a more general name.

Reply:
Maybe just `$T`? It just functions as an injection point for type information, right?